services/integration.yaml

```diff
@@ -3,6 +3,24 @@ brig:
   host: 127.0.0.1
   port: 8082
 
+# The brigs that live in the other federations. Talk to it directly for testing
+# purposes
+federatedBrig1:
+  host: 127.0.0.2
+  port: 8082
+
+federatedBrig2:
+  host: 127.0.0.3
+  port: 8082
+
+federator1:
+  host: 127.0.02
```

Comment:
Typo.

AIso, is there no federator on `127.0.0.1`? We would need it, if the main instance is supposed to participate in the federated tests.

Reply:
Good catch and good catch! :)